Applications/Spire/Include/Spire/SpireTester/SpireTester.hpp

```diff
@@ -29,6 +29,12 @@ namespace Spire {
       });
     app.exec();
   }
+
+  /**
+   * Processes events in the Qt event loop until the given predicate returns
+   * true.
+   */
+  void wait_until(const std::function<bool ()>& predicate);
```

Comment:
Changes to the public API need to be approved. Remove this function.

Reply:
I'm looking into a TestOrderImbalanceIndicatorModel than can resolve subscriptions manually.